terraform/modules/sync_check/service/init.sh

```diff
@@ -30,8 +29,8 @@ metrics_nfs_sample_rate: 600
 container_cache_metadata_limit: 600
 disable_zero_mem_process_filter: true
 disable_all_plugins: true
-disable_cloud_metadata: true 
-ignore_system_proxy: true 
+disable_cloud_metadata: true
+ignore_system_proxy: true
```

Comment:
nonblocking: you can add a formatter to ensure there's no trailing whitespace so changes like these aren't needed